libs/expo/betterangels/src/lib/screens/AddEditClient/PersonalInfo/HmisProfiles.tsx

```diff
@@ -5,9 +5,10 @@ import {
   Radio,
```

Comment:
Hi @tglaz 

I confirmed I see the error message when I tried to submit without the HMIS ID#. 
I read the comment in the JIRA ticket regarding the AC#1. When are you planning to fix the part? 
Let me know! Thanks!